views/index.erb

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+  <head>
+    <title>MakersBNB</title>
+  </head>
+  <body>
+    <form action='/users' method='post'>
```

Comment:
action='/session/signup' maybe?

Reply:
Agreed - can we plan more tomorrow regarding the sign up behaviour?